release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3322,6 +3322,39 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.14.28
+[id="ocp-4-14-28"]
+=== RHSA-2024:3523 - {product-title} 4.14.28 bug fix update and security update
```

Comment:
```suggestion
=== RHSA-2024:3523 - {product-title} 4.14.28 bug fix and security update
```
IMHO don't need "update" twice; not a merge-stopper, just a thought

Reply:
Thanks, Shuana. I'll update as part of the 4.14.29 release notes, which I'm also documenting. Thank you!